### DISCUSSION TITLE
What would be needed to make it work with @sveltejs/adapter-cloudflare

### DISCUSSION THREAD

**Discussion by fev4** [icflorescu/trpc-sveltekit#44] [Q&A]
Hey!

First off, thanks for creating trpc-sveltekit!

I've fallen in love with svelte-kit since it's inception and I've been following Theo (ping.gg) for a while and finally decided to give tRPC a try, but found out they don't natively support SvelteKit. But then I found this!

I would love to be able to use it, but for our app we are currently using cloudflare, which according to the homepage, it's not supported just yet.

What would need to happen for it to be supported as well?

**Comment by icflorescu** [comment]
First off, thanks for your kind words!
Obviously, I share your feelings towards SvelteKit, though React is currently paying my bills :-)

> What would need to happen for it to be supported as well?
 
 I think it should work with cloudflare as it is.
 The internals are pretty similar to the basic `fetch` adapter, so I see no reason why it shouldn't work with cloudflare, or even with deno.

Actually, I'd be really grateful if you could try it with cloudflare if you have a test environment ready and let me know how things work so I can update the docs 🙏

Shameless plug:
Feel free to drop me a message if there's work I can help you with.
I'm fairly good with React (see my GitHub [profile](https://github.com/icflorescu), and I'm also the author of [Mantine DataTable](https://icflorescu.github.io/mantine-datatable/)), but - as I said above - I share your feelings towards Svelte/SvelteKit (hence tRPC-SvelteKit) and I'd be really happy to work on projects built on these technologies in the future.

**Comment by icflorescu** [comment]
Any feedback on this, @fev? Did you manage to test it with cloudflare?

**Reply by fev4** [reply]
Hey there! I totally missed the initial response! Sorry about that. 🙏 
I can give this a try during the week and I'll comment back. Give me a few days.

And thanks for the plug! If I find myself in a position to hire someone or recommend anyone for React/Svelkit you'll be the first to know! Right now the Sveltekit app is more like a personal project (for 2 guys and me), very small and just getting started. And in my current job I'm a data engineer where our website is built in Wordpress, and AFAIK the CEO is doing a hiring freeze. So at this very moment I don't have a path to recommend to anyone really.

**Reply by icflorescu** [reply]
Not a problem, it looks like I'll be very busy till July 2023 anyways :-)
Do let me know if it works fine on cloudflare.

**Reply by Manubi** [reply]
@fev4 do we have any news on this one? :)
